internal/v2/cert_verifier/cert_verifier_test.go

```diff
@@ -35,39 +80,74 @@ func TestVerifyClientCertChain(t *testing.T) {
 	for _, tc := range []struct {
 		description              string
 		pool                     *x509.CertPool
-		expectedCommonName       string
 		rawCerts                 [][]byte
 		verifiedChains           [][]*x509.Certificate
-		err                      error
+		err			 error
 	}{
 		{
 			description: "static : empty certPool",
 			pool: certPool0,
-			expectedCommonName: "s2a_test_cert",
 			rawCerts: nil,
 			verifiedChains: nil,
-			err: nil,
+			err: errors.New("rpc error: code = Unknown desc = Client Peer Verification failed: client cert chain is empty."),
 		},
 		{
 			description: "static : non-empty certPool",
 			pool: certPool1,
-			expectedCommonName: "s2a_test_cert",
-			rawCerts: cert1.Certificate,
+			rawCerts: [][]byte{clientDERCert,},
```

Comment:
Do we have any tests where more than 1 cert is sent? If not, please add them. :)

Reply:
Will do! I create a chain of length 2 : root_cert.der which is self signed, and leaf_cert.der which is signed by root_cert.der.